app/src/pages/project/DocumentEvaluationSummary.tsx

```diff
@@ -23,14 +24,21 @@ type DocumentEvaluationSummaryProps = {
 export function DocumentEvaluationSummary({
   evaluationName,
 }: DocumentEvaluationSummaryProps) {
+  const { projectId } = useParams();
   const data = useLazyLoadQuery<DocumentEvaluationSummaryQuery>(
     graphql`
-      query DocumentEvaluationSummaryQuery($evaluationName: String!) {
-        ...DocumentEvaluationSummaryValueFragment
-          @arguments(evaluationName: $evaluationName)
+      query DocumentEvaluationSummaryQuery(
+        $evaluationName: String!
+        $id: GlobalID!
```

Comment:
```suggestion
        $projectId: GlobalID!
```
could make it just a tad more clear. nbd